docs/en/development/build-osx.md

```diff
@@ -5,7 +5,7 @@ toc_title: How to Build ClickHouse on Mac OS X
 
 # How to Build ClickHouse on Mac OS X {#how-to-build-clickhouse-on-mac-os-x}
 
-Build should work on Mac OS X 10.15 (Catalina)
+Build should work on Mac OS X 10.15 (Catalina).
```

Comment:
Here it is no russian translation for the whole file docs/ru/development/build-osx.md

Reply:
Yes. We don't need to translate it.
Marker of the situations is:
```
$ ls -l docs/ru/development/build-osx.md 
lrwxrwxrwx 1 bayonet bayonet 33 мая  3 22:08 docs/ru/development/build-osx.md -> ../../en/development/build-osx.md
```
The file is a symbolic link.